src/app/header/header.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
```

Comment:
seems like that file can be replaced with inline html as it is really small

Reply:
This file will be filled in next PR